apps/codeforafrica/src/lib/api.netlify-cms/sections/getSEO.js

```diff
@@ -0,0 +1,11 @@
+import { join } from "path";
```

Comment:
1. `getSeo`
2. Move it to `lib/api.netlify-cms`. Not sure what sections add to it.